### DISCUSSION TITLE
How to specify the source colorspace when converting images

### DISCUSSION THREAD

**Discussion by tobyliu415** [ImageMagick/ImageMagick#3191] [General] [answered]
I found the same TIFF image.
on im6 ->
TIFF 4300x3040 4300x3040+0+0 8-bit TrueColor sRGB
on im7 ->
TIFF 4300x3040 4300x3040+0+0 8-bit TrueColor YCbCr

Then im6 works, im7 outputs with wrong color.

The command line doesn't let im7 treat the input TIFF as sRGB.
magick convert -verbose -colorspace RGB 20210202155246.tif d.png

How to specify the source colorspace ?

**Comment by fmw42** [comment]
In IM 7, use magick, not magick convert and not convert.  For other functions use magick identify, magick montage, etc. See if that works.

**Comment by tobyliu415** [comment, marked as answer]
Removing keyword convert does not help.
But it works when I tried :
magick -verbose ../images/20210202133204.tif -set colorspace RGB a.png
